configure.ac

```diff
@@ -394,6 +394,8 @@ then
             if test "$ENABLED_FIPS" = "no"
             then
                 test "$enable_openssh" = "" && enable_openssh=yes
+                # S/MIME support requires PKCS7, which requires no FIPS.
+                test "$enable_smime" = "" && enable_smime=yes
```

Comment:
Had to move this line.  Otherwise, --enable-all + --enable-sp-math was also enabling S/MIME, which in turn enabled openSSL ALL and blew past our stack limits.